core/src/com/mygdx/game/models/BrainstormingPhase.java

```diff
@@ -0,0 +1,59 @@
+package com.mygdx.game.models;
+import java.util.ArrayList;
+
+/**
+ * The BrainstormingPhase keeps track of the brains that have been fired and the wall.
+ *
+ * BRAIN_DAMAGE: How much damage 1 brain does to the wall
+ * Brains: The list of brains that has been sendt to the wall
+ * Wall: The wall
+ *
+ * This class implements the MVC pattern as a model.
+ *
+ * */
+
+public class BrainstormingPhase {
+
+    private final int BRAIN_DAMAGE;
+    private final Wall wall;
+    private ArrayList<Brain> brains = new ArrayList<>();
+    private final Player player;
+
+    /**
+     * The constructor that takes the hitpointmax of the wall, and
+     * how much damage a brain will do
+     * @param maxHitPoints: Walls max HP.
+     * @param BRAIN_DAMAGE: The amount of damage one brain does.
+     */
+    public BrainstormingPhase(Player player,int maxHitPoints, int BRAIN_DAMAGE){
```

Comment:
```suggestion
    public BrainstormingPhase(Player player, int maxHitPoints, int BRAIN_DAMAGE){
```
Vary minor change. Just added a space.